contributor-guide/modules/ROOT/pages/testing/continuous-integration.adoc

```diff
@@ -130,20 +142,175 @@ CircleCI also supports parallel testing, caching of dependencies, and matrix bui
 
 Refer to:
 
-* https://github.com/boostorg/boost/blob/master/.circleci/config.yml[Super-project config.yml]
 * https://github.com/boostorg/beast/blob/c316c6bd3571991aeac65f0fc35fca9067bc7906/.circleci/config.yml[Beast library config.yml]
 * https://github.com/boostorg/geometry/blob/2ec9d65d1294edb97157b564726fdf56b6ac562f/.circleci/config.yml[Geometry library config.yml]
 * https://github.com/boostorg/multiprecision/blob/380aae3c28c646ea2ca1b42156d83732295082d7/.circleci/config.yml[Multiprecision library config.yml]
 
-=== CircleCI Reference
+== Codecov.io
```

Comment:
I think coverage deserves its own section, and that we should expand on how to set it up correctly.

Reply:
Created a coverage section, with a new intro.